template-for-proof/README.md

```diff
@@ -8,3 +8,14 @@ To run the proof.
   to your path.
 * Run "make".
 * Open html/index.html in a web browser.
+
+Getting started with Makefiles using `arpa`
+-------------
+
+The [`arpa`](https://github.com/awslabs/aws-proof-build-assistant) tool helps you get started with writing proof Makefiles.
+
+To use `arpa`.
+* Run "make arpa" to generate a Makefile.arpa that contains relevant build information for the proof.
+* Use Makefile.arpa as the starting point for your proof Makefile by:
+  1. Modifying Makefile.arpa (if required).
+  2. Including Makefile.arpa into the existing proof Makefile.
```

Comment:
Want to give the line "sinclude Makefile.arpa" to add to the Makefile and suggest where (top or bottom) to put the line?